crowbar_framework/lib/crowbar/deployment_queue.rb

```diff
@@ -375,7 +375,9 @@ def elements_not_ready(nodes, pre_cached_nodes = {})
         next if node.nil?
 
         pre_cached_nodes[n] = node
-        delay << n if node.crowbar["state"] != "ready" and !delay.include?(n)
+        # allow commiting proposal for nodes in the crowbar_upgrade state
+        state = node.crowbar['state']
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)